utils/search/search_general.py

```diff
@@ -46,7 +46,7 @@ def convert_to_solr_document(sound):
     document["tag"] = getattr(sound, "tag_array")
     document["license"] = getattr(sound, "license_name")
 
-    if getattr(sound, "pack_id"):
+    if getattr(sound, "pack_name"):
```

Comment:
Why this change? Is this a bug that was in here before?

Reply:
Ok I understand it now, this is because the updated query will set `pack_name` to null if pack has been deleted therefore we need to check on `pack_name` to decide whether to index the pack field or not.

Reply:
yes, that is the case